### PR TITLE
Add RabbitMQ service to Docker Compose for message queue integration

### PR DESCRIPTION
**Pull request by jah3**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: notifications_db
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  email-service:
```
Comment (taragrammatonn) [inline docker-compose.yaml:26]: This should be done by Mihail

**Review by taragrammatonn — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (jah3) [reply docker-compose.yaml:26]: You mean email service docker?

Comment (taragrammatonn) [reply docker-compose.yaml]: all related email service thigs

**Review by taragrammatonn — CHANGES_REQUESTED** [review]

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    environment:
+      - RABBITMQ_DEFAULT_USER=guest
+      - RABBITMQ_DEFAULT_PASS=guest
+    volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
+
+volumes:
```
[the remark this answers is not in the document]
Comment (jah3) [reply docker-compose.yaml:15]: kinda, it ensures that after several operations the data wouldn't be lost

**Comment by taragrammatonn** [comment]
pls edit commit message

**Pull request closed without merging**